addons/linkify-mentions/addon.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Clickable forum mentions",
+  "description": "Turns @username mentions and quote authors on the forums into clickable links.",
+  "credits": [
+    {
+      "name": "Samq64",
+      "link": "https://github.com/Samq64"
+    },
+    {
+      "name": "just-a-hriday",
+      "link": "https://scratch.mit.edu/users/just-a-hriday"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "script.js",
+      "matches": ["topics"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "plain-text.css",
+      "matches": ["topics"],
+      "if": {
+        "settings": {
+          "plainText": true
+        }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "name": "Style quote authors as normal text",
```

Comment:
Should this also apply to mentions?

Reply:
> Should this also apply to mentions?

Nah, it would be even more confusing whether it was text or a link if it was part of an actual sentence.